tools/scripts-core/src/scripts/start-storybook.js

```diff
@@ -5,10 +5,10 @@ import { getStorybookConfiguration } from '../utils/storybook.js';
 
 export default async function start(env, presetApi, options) {
 	utils.pkg.checkPackageIsInstalled('@talend/scripts-config-storybook-lib');
-	const startStorybook = utils.path.resolveBin('start-storybook');
+	const startStorybook = utils.path.resolveBin('storybook');
 	const sbConfigPath = getStorybookConfiguration(presetApi);
 
-	return utils.process.spawn(startStorybook, ['-c', sbConfigPath].concat(options), {
+	return utils.process.spawn(startStorybook, ['dev', '-c', sbConfigPath].concat(options), {
```

Comment:
`build` and `start` are doing the same now ? We keep them both for compat purposes or am I missing something ?

Reply:
afaik they're just using the same bin (`storybook` line 8), and the only thing that differs is a parameter in bin (`dev` to start and `build` to build) 🤷